docs/en/docs/deployment/server-workers.md

```diff
@@ -90,7 +90,7 @@ Let's see what each of those options mean:
         ```
 
     * So, the colon in `main:app` would be equivalent to the Python `import` part in `from main import app`.
-* `--workers`: The number of worker processes to use, each will run a Uvicorn worker, in this case, 4 workers.
+* `--workers`: The number of worker processes to use, each will run an Uvicorn worker, in this case, 4 workers.
```

Comment:
Hmm how do you pronounce Uvicorn?
I always pronounced it like `Universe` and it's `a Universe` not `an Universe`